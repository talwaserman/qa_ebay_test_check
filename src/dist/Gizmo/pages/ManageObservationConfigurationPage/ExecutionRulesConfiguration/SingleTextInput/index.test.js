"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));

var _index = _interopRequireDefault(require("./index"));

it('SingleTextInput - SingleTextInput when currentData is empty', function () {
  var tree = _reactTestRenderer.default.create( /*#__PURE__*/_react.default.createElement(_index.default, {
    inputLabel: 'testLabel',
    currentInputData: {
      name: null,
      rowId: null
    }
  })).toJSON();

  expect(tree).toMatchSnapshot();
});
it('SingleTextInput - SingleTextInput when currentData is with Data', function () {
  var tree = _reactTestRenderer.default.create( /*#__PURE__*/_react.default.createElement(_index.default, {
    inputLabel: 'testLabel',
    currentInputData: {
      name: 'name1',
      rowId: 'rowId1'
    }
  })).toJSON();

  expect(tree).toMatchSnapshot();
});