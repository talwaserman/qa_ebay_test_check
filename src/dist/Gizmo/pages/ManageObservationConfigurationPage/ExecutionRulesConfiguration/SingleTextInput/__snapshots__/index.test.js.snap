// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SingleTextInput - SingleTextInput when currentData is empty 1`] = `
<div
  className="single-text-input-wrapper"
>
  <div
    className="input-label"
  >
    testLabel
  </div>
  <div
    className="configuration-input"
  >
    <input
      className="ant-input"
      onChange={[Function]}
      onKeyDown={[Function]}
      type="text"
      value=""
    />
  </div>
  <br />
  <br />
  <br />
  <br />
  <br />
  <br />
  <br />
</div>
`;

exports[`SingleTextInput - SingleTextInput when currentData is with Data 1`] = `
<div
  className="single-text-input-wrapper"
>
  <div
    className="input-label"
  >
    testLabel
  </div>
  <div
    className="configuration-input"
  >
    <input
      className="ant-input"
      onChange={[Function]}
      onKeyDown={[Function]}
      type="text"
      value="name1"
    />
  </div>
  <br />
  <br />
  <br />
  <br />
  <br />
  <br />
  <br />
</div>
`;
